### PR TITLE
update outdated terms in "gas and fees" article

### PR DESCRIPTION
**Pull request by umededoteth**
## Description

Hi team! Just thought I'll update some terms. PoW had miners. PoS has validators. 

Let me know if there's anything wrong with these corrections!

### CODE REVIEW

**Comment by nhsz** [comment]
@wackerow would like your input here

File: src/content/developers/docs/gas/index.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ It's also important to note it is unlikely we will see extended spikes of full b
 
 ### Priority fee (tips) {#priority-fee}
 
-Before the London Upgrade, miners would receive the total gas fee from any transaction included in a block.
+Before the London Upgrade, validators (known as "miners" under PoW) would receive the total gas fee from any transaction included in a block.
```
Comment (corwintines) [inline src/content/developers/docs/gas/index.md:86]:
```suggestion
Before the London Upgrade, validators (previously "miners" under PoW) would receive the total gas fee from any transaction included in a block.
```

**Review by corwintines — COMMENTED** [review]
Just one small change that I think improves this

**Comment by corwintines** [comment]
@all-contributors can you add @umededoteth for content

**Comment by corwintines** [comment]
Wanted to apologize for this not getting in @umededoteth

I see this update was done https://github.com/ethereum/ethereum-org-website/pull/10636 

Going to close this out, but appreciate you opening this. Hope to see more contributions in the future, will be more on top of this.

**Pull request closed without merging**